src/3ds.ts

```diff
@@ -1,26 +1,34 @@
 import crypto from 'node:crypto';
 import { md5, getDataFromPathOrBuffer } from '@/util';
 
-export type CTRBOSSContainer = {
-	hash_type: number;
-	release_date: bigint;
-	iv: Buffer;
-	content_header_hash: Buffer;
-	content_header_hash_signature: Buffer;
+export type CTRPayloadContent = {
 	payload_content_header_hash: Buffer;
 	payload_content_header_hash_signature: Buffer;
 	program_id: bigint;
 	content_datatype: number;
 	ns_data_id: number;
+	version: number;
 	content: Buffer;
 }
 
+export type CTRBOSSContainer = {
+	hash_type: number;
+	// TODO - This isn't a release date! See https://www.3dbrew.org/wiki/SpotPass#BOSS_Header
```

Comment:
If it's not a release date why is it named as such?

Reply:
I had kept the original name it had. I can rename it to `serial_number` for consistency with 3dbrew though